src/app/content/highlights/components/SummaryPopup/Filters.tsx

```diff
@@ -97,7 +97,8 @@ export default () =>
       <ConnectedPrintButton />
     </FiltersTopBar>
     <ConnectedFilterList
-      colorAriaLabelKey={() => 'i18n:highlighting:filters:remove:color'}
+      colorAriaLabelKey='i18n:highlighting:filters:remove:color'
```

Comment:
this was a function to unify type with `colorLabelKey`
I think you should add here also `analyticsLabel` props that should also accept a function  with `color` parameter and return something based on this